### ISSUE TITLE
Not working on DS216j (Armada 385) and 9.1.3-10869

### ISSUE THREAD

**Issue by DDAAXX** [Kaitiz/Surveillance-Station-9#14]
Hi,
I tried just now and after install, the Surveillance Station became unreachable.
I uninstalled and reinstalled from package manager to have it back.

Do you plan to support this?
Thanks!!!

**Comment by AmmokPT** [comment]
not working on 9.1.3-10869

**Comment by dodavies** [comment]
DS415+ 
works for me 9.1.3-10869
DSM 7.1.1-42962

**Comment by catfordfire** [comment]
same. rolled back to previous version, disabled auto-updates, and it's fine.

**Comment by javifly** [comment]
and you haven't ordered the configurations?How did you do the process, uninstall 9.1.3 and install the old one?

**Comment by DDAAXX** [comment]
Yes, I did the same: I uninstalled the 9.1.3 and installed the one available on this github. I also disabled the auto-update function in order to avoid problem.

**Comment by sania-owner** [comment]
You can use cheap licenses that allow you to update any version of SurveillanceStation
https://bc.saniaowner.space/

**Comment by catfordfire** [comment]
sure, but for not much more, I can buy a legit one.

**Comment by iCr** [comment]
Surveillance Station 9.1.2-10854 does not working too at 216j. It opens blank page only.
